src/smc-webapp/editor.coffee

```diff
@@ -1932,13 +1932,33 @@ class CodeMirrorEditor extends FileEditor
         for i, cm of [@codemirror, @codemirror1]
             size = @local_storage("font_size#{i}")
             if size?
-                @set_font_size(cm, size)
+                if size < 2
+                    size = 2
+                @set_font_size(cm, size, 0)
             else if @default_font_size?
-                @set_font_size(cm, @default_font_size)
+                @set_font_size(cm, @default_font_size, 0)
 
-    set_font_size: (cm, size) =>
+    set_font_size: (cm, size, delta) =>
         if size > 1
             elt = $(cm.getWrapperElement())
+            if delta != 0
+                for html, i in elt.find('.sagews-output-html').children()
```

Comment:
so, my first instinct here is, that since this is jquery, you do not have to iterate. just specify in the find-query that you mean all the children and then run an operation once on all of them.